packages/checkbox/src/index.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from "react"
+import styled from "styled-components/macro"
+import {Text} from "@smashing/typography"
+import {getCheckboxStyle, getLabelStyle} from "./styles"
+import {useDefaults} from "@smashing/theme"
+import {CheckboxAppearanceType, CheckboxProps, StyledTextProps} from "./types"
+
+const Label = styled(Text)<StyledTextProps>`
+  display: flex;
+  align-items: center;
+  margin: 0;
+  cursor: ${_ => (_.disabled ? "not-allowed" : "pointer")};
+  ${_ => getLabelStyle(_.appearance, _.disabled, _.checked)};
+  svg {
```

Comment:
This can cause some issues if you have more than one svg in card. This style should be bound directly to the svg. In story you can just use inline style